packages/components/src/radio/tests/chromatic/RadioGroup.stories.tsx

```diff
@@ -0,0 +1,238 @@
+import type { Meta, StoryObj } from "@storybook/react";
+
+import { ErrorMessage } from "../../../errorMessage/index.ts";
+import { HelperMessage } from "../../../helperMessage/index.ts";
+import { Label } from "../../../Label/index.ts";
+import { Inline, Stack } from "../../../layout/index.ts";
+import { Text } from "../../../Text/index.ts";
+import { Radio } from "../../src/Radio.tsx";
+import { RadioField } from "../../src/RadioField.tsx";
+import { RadioGroup } from "../../src/RadioGroup.tsx";
+import { RadioList } from "../../src/RadioList.tsx";
+
+const meta = {
+    title: "Components/Radio/RadioGroup",
+    component: RadioGroup
+} satisfies Meta<typeof RadioGroup>;
+
+export default meta;
+
+type Story = StoryObj<typeof meta>;
+
+export const Vertical = {
+    render: props => (
+        <Stack>
```

Comment:
you should add a couple of tests with icons

Reply:
Added, let me know if it's as expected